lib/generators/partisan/templates/migration.rb

```diff
@@ -9,6 +9,7 @@ def up
 
     add_index :follows, ['follower_id', 'follower_type'],     name: 'index_partisan_followers'
     add_index :follows, ['followable_id', 'followable_type'], name: 'index_partisan_followables'
+    add_index :follows, ['follower_id', 'follower_type', 'followable_id', 'follower_type'], name: 'index_partisan_unique_follow', unique: true
```

Comment:
Since lines before this one have `name:` with extra whitespace for alignment, should we change the previous lines to fix align or remove extra spaces?